geomesa-tools/src/test/scala/org/locationtech/geomesa/tools/ConvertCommandTest.scala

```diff
@@ -215,14 +215,9 @@ class ConvertCommandTest extends Specification with LazyLogging {
     FileUtils.deleteQuietly(testCSVFile)
     FileUtils.deleteQuietly(testJSONFile)
     FileUtils.deleteQuietly(testTSVFile)
-    FileUtils.deleteQuietly(testCSVFile)
-    FileUtils.deleteQuietly(testCSVFile)
-    FileUtils.deleteQuietly(testCSVFile)
-    FileUtils.deleteQuietly(testCSVFile)
-    FileUtils.deleteQuietly(testCSVFile)
```

Comment:
Without this, how will know that the file has been *really, really, really* deleted?